low-level/files/opt/seagate/sspl/conf/sspl.prepare.conf.tmpl

```diff
@@ -0,0 +1,26 @@
+server_node:
```

Comment:
this needs to be super-set of post-install cfg template. I see release->product missing

Reply:
Done.